test/test_torch.py

```diff
@@ -195,8 +195,6 @@ def test_namespace(ns, *skips):
             test_namespace(torch.randn(1),
                            'as_strided_',
                            re.compile('^clamp_(min|max)_?$'),
-                           'coalesce',
-                           'is_coalesced',
```

Comment:
These test changes seem unrelated to a docs update. If they are needed, put them in a separate PR instead?

Reply:
Actually, this is related docs updates as otherwise, the doc tests will fail. Basically, when one defines a docstring for `coalesce` and others, one needs to remove the corresponding `test_namespace` arguments.
So I would keep this changeset within this PR.